2.4/blocks/aspirelists/db/tasks.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+$tasks = array(
```

Comment:
Can you confirm that 4am times are converted to local time on the Moodle Server?  We wouldn't want APAC customers doing something at 4am GMT :) .

Reply:
Yes - it doesn't appear to be specified in the docs but a read of scheduled_task class shows it's using the server's local timezone.